recipes/mqc/run_test.pl

```diff
@@ -0,0 +1,11 @@
+# Test file for testing Perl modules installation
+$|=1;
```

Comment:
Easiest solutions to me seems to use the "tests: imports:" for python packages and the "test: commands: perl -e "use Parallel::ForkManager;" for perl packages